spec/models/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    USER_FIELDS = %i(email country).freeze
+    subject { FactoryGirl.build(:user) }
+
+    USER_FIELDS.each do |attr|
```

Comment:
Since we're only using these fields once, we don't need to set a `USER_FIELDS` constant above. Instead we can refer to the params directly in this each block.